kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -129,10 +129,14 @@ def wiki_to_html(
 @library.filter
 def wiki_to_safe_html(wiki_markup, locale=settings.WIKI_DEFAULT_LANGUAGE, nofollow=True):
     """Wiki Markup -> HTML Markup object with limited tags"""
-    html = parser.wiki_to_html(wiki_markup, locale=locale, nofollow=nofollow)
-    return Markup(
-        bleach.clean(html, tags=ALLOWED_BIO_TAGS, attributes=ALLOWED_BIO_ATTRIBUTES, strip=True)
+    html = parser.wiki_to_html(
+        wiki_markup,
+        locale=locale,
+        nofollow=nofollow,
+        tags=ALLOWED_BIO_TAGS,
+        attributes=ALLOWED_BIO_ATTRIBUTES,
     )
+    return Markup(html)
```

Comment:
We realized later that the extra `bleach.clean` call was necessary because it strips rather than escapes, and there's no way to pass `strip=True` into the `py-wikimarkup` parser's `parse` method.